rasa/core/channels/channel.py

```diff
@@ -23,7 +23,18 @@
 class UserMessage(object):
     """Represents an incoming message.
 
-     Includes the channel the responses should be sent to."""
+     Includes the channel the responses should be sent to.
+
+     Attributes:
+        text (Text): The message text content.
+        output_channel (OutputChannel): The message output data.
+        sender_id (Text): The message owner ID.
+        parse_data (Text, Any): Rasa data about the message.
+        input_channel (Text): The message origin name.
+        message_id (Text): ID of the message.
+        metadata (Dict): A dict that contains information about the message.
```

Comment:
can we move this part a docstring for the constructor?

Reply:
as said in the call I'd also remove the types since that's already documented in the code through the type definitions